draft-ietf-jsonpath-base.md

```diff
@@ -381,7 +381,7 @@ of the evaluation of a filter expression ({{filter-selector}}).
 
 ### Segments
 
-Segments select children (`[]`) or descendants (`..[]`) of an input value.
+Segments select children (`[<selectors>]`) or descendants (`..[<selectors>]`) of an input value.
```

Comment:
The only problem I have is that this amplifies a bit that segments are always bracket notation, which they aren't.

Reply:
> segments are always bracket notation

I read this more that brackets are more of a formal notation.  Shorthand notation just works around that.

Reply:
> I take your point, but actually the existing text does the same. We could add "e.g., " inside each of the parentheses, but that seems a bit fussy.

Indeed.  I just wanted to mention this observation, but it is not in the way of merging this PR.